src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java

```diff
@@ -193,7 +193,7 @@ public static EnvType getEnvType(String env) {
     } else if (env.contains("staging")) {
       return EnvType.STAGING;
     } else if (env.contains("verify") || env.contains("preview") || env.contains("sandbox")
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1067)